modules/table/src/test/java/org/apache/ignite/internal/table/distributed/AbstractMvStorageUpdateHandlerTest.java

```diff
@@ -131,7 +131,7 @@ void tearDown() throws Exception {
 
     @Test
     void testConcurrentExecuteBatchGc() {
-        assertThat(distributionZoneConfig.dataStorage().gcOnUpdateBatchSize().update(2), willSucceedFast());
+        assertThat(distributionZoneConfig.dataStorage().gcOnUpdateBatchSize().update(4), willSucceedFast());
```

Comment:
Since **tryLock** is now used, there may be a situation where we did not delete all after trying to replay the race, so we should increase the size of the batch.